_articles/dash-to-dock.md

```diff
@@ -42,27 +42,27 @@ On the resulting pop-up, click Install to download and install the extension.
 
 ![Enabling Dash to Dock](/images/dash-to-dock/pop-install-extension.png)
 
-At this point, Firefox can be closed. The launcher will now display on the left side of the screen, similar to Ubuntu.
```

Comment:
The reason I wrote "similar to Ubuntu" when I re-did this article was because I've had people say they chose Ubuntu over Pop!_OS because they don't have to open the Activities menu to see the launcher, just FYI.

Reply:
Thank you for confirming the reasoning. We can add it back.